src/app.js

```diff
@@ -1,3 +1,35 @@
+/* eslint-disable no-console */
 'use strict';
 
-// Write your code here
+const readline = require('readline');
+const { generateRandomNumber } = require('./modules/generateRandomNumber.js');
+const { getBullsAndCows } = require('./modules/getBullsAndCows.js');
+const { checkIsValidUserInput } = require('./modules/checkIsValidUserInput.js');
+
+const numberToGues = generateRandomNumber();
```

Comment:
There is a typo in the variable name 'numberToGues'. It should be 'numberToGuess' to correctly reflect its purpose.